common-util/src/main/java/org/sunbird/services/sso/impl/KeyCloakServiceImpl.java

```diff
@@ -41,19 +41,20 @@
  * @author Manzarul
  */
 public class KeyCloakServiceImpl implements SSOManager {
-
+	
+  
   private Keycloak keycloak = KeyCloakConnectionProvider.getConnection();
   private static final String URL =
       KeyCloakConnectionProvider.SSO_URL
           + "realms/"
           + KeyCloakConnectionProvider.SSO_REALM
           + "/protocol/openid-connect/token";
-  private static final String SSO_PUBLIC_KEY = System.getenv(JsonKey.SSO_PUBLIC_KEY);
-
+ // private static final String SSO_PUBLIC_KEY = System.getenv(JsonKey.SSO_PUBLIC_KEY);
```

Comment:
Please do not comment out code, please remove it entirely.

Reply:
okay, i will do the respective changes